engine/src/flutter/shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -332,9 +331,7 @@ private FlutterEngineGroup.Options addEntrypointOptions(FlutterEngineGroup.Optio
             + " this FlutterFragment.");
 
     FlutterEngineGroup group =
-        engineGroup == null
-            ? new FlutterEngineGroup(host.getContext(), host.getFlutterShellArgs().toArray())
-            : engineGroup;
+        engineGroup == null ? new FlutterEngineGroup(host.getContext()) : engineGroup;
```

Comment:
Does this still work?  I would have expected the enforcement to be happening at a different layer.  Now this isn't forwarding any flags to the group.